integration/deployment/default.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: danm.k8s.io/v1
+kind: DanmNet
+metadata:
+  name: default
+  namespace: kube-system
+spec:
+  NetworkID: default
+  NetworkType: flannel
```

Comment:
where is the Flannel DaemonSet, and the Flannel network config installed? I can't find it in the review

either we should take care of installing everything on our own, or we should not configure Flannel as the default backend